portal/src/graphql/adminapi/ChangePasswordScreen.tsx

```diff
@@ -131,19 +131,21 @@ const ChangePasswordContent: React.VFC<ResetPasswordContentProps> = function (
         onSubmit={onSubmit}
         noValidate={true}
       >
-        <div>
-          <TextField
-            label={renderToString("ChangePasswordScreen.email")}
-            type="email"
-            value={user?.standardAttributes?.email ?? ""}
-            disabled={true}
-          />
-          <ChoiceGroup
-            selectedKey={state.passwordCreationType}
-            options={passwordCreateionTypeOptions}
-            onChange={onChangePasswordCreationType}
-          />
-        </div>
+        {user?.standardAttributes?.email != null ? (
+          <div>
+            <TextField
+              label={renderToString("ChangePasswordScreen.email")}
+              type="email"
+              value={user.standardAttributes.email}
```

Comment:
Condition to show this:

1. When the user has a email Login ID. Having a email standard attribute is not enough.